koku/api/report/ocp/provider_map.py

```diff
@@ -208,6 +216,10 @@ def __init__(self, provider, report_type):
                             "cost_total": self.cloud_infrastructure_cost_by_project
                             + self.markup_cost_by_project
                             + self.cost_model_cost,
+                            "cost_distributed": self.cloud_infrastructure_cost_by_project
```

Comment:
What do you think about `cost_total_distributed` ?

Reply:
it seems more consistent, thanks 😀 
I've updated the provider map